### PR TITLE
Ticket 190 upgrade to django 3

### LINKED ISSUE #190: Make VNG API Common Django 3.2 compatible
Currently vng api common is pinned to < Django 3.0, but 2.2 is end of life in April this year and this library currently blocks upgrading CMIS adapter and Open Zaak to 3.2 LTS.

**Tasks**

- [ ] Update the `tox.ini` envlist to support the following matrix of versions
    - Python 3.7 - Django 2.2
    - Python 3.8 - Django 2.2
    - Python 3.7 - Django 3.2
    - Python 3.8 - Django 2.2
- [ ] Update the Github CI workflow to support the same matrix
- [ ] Make the necessary changes to the package to support Django 3.2
- [ ] Update the package metadata to correctly reflect the supported versions
- [ ] Extra: see if we can add Python 3.9 and Python 3.10 support

### PR DESCRIPTION
**Pull request by ahmadpoorgholam**
fixes #190

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -20,7 +20,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        env: ["isort", "docs", "py36-django22"]
+        python: ['3.6', '3.7', '3.8' ]
+        django: [ '2.2', '3.2' ]
```
Comment (sergei-maertens) [inline .github/workflows/ci.yml:24]: good improvement, but we lost tox envs `isort` and `docs` here.

can you:

*  add a separate docs workflow job like in Open Zaak: https://github.com/open-zaak/open-zaak/blob/main/.github/workflows/ci.yml#L180 and 
* add a code_quality.yml workflow that runs at least isort, you can use https://github.com/open-zaak/open-zaak/blob/main/.github/workflows/code_quality.yml for inspiration

**Comment by ahmadpoorgholam** [comment]
I will add "isort", "docs" to the tox command soon

File: .github/workflows/ci.yml (hunk 1 above)
Comment (sergei-maertens) [inline .github/workflows/ci.yml:24]: Additionally, please update the `name` of the job which now still references the `${{ matrix.env }}` which is no longer defined.

File: setup.cfg [hunk 2]
```diff
@@ -34,7 +34,7 @@ scripts =
   bin/use_external_components
   bin/use_external_components.cmd
 install_requires =
-    django~=2.2.0
+    django < 4.0
```
Comment (sergei-maertens) [inline setup.cfg:37]: I'd still require a minimum version of Django though

```suggestion
    django>=2.2,< 4.0
```

**Review by sergei-maertens — CHANGES_REQUESTED** [review]
There are a lot of warnings too when running tests with Django 3.2 - can you resolve those? They will make upgrading to Django 4.x at some point easier.

**Review by sergei-maertens — CHANGES_REQUESTED** [review]
I'm also missing the required import/code change that breaks cmis-adapter.

The vng-api-common test-suite is not extensive enough and that codepath is not covered, clearly.

**Comment by sergei-maertens** [comment]
Doing this in #193

**Pull request closed without merging**